notes.org

```diff
@@ -37,24 +37,29 @@
 		 let posts_in_blog = query!(Post, &conn, blog = {blog}).load(&conn)
 	 }
 	 #+END_SRC
+
 	 struct Posts
+
+	 #+BEGIN_SRC sql
```

Comment:
do we need this, or the Rust above?

i.e. is anything in the above missing from butane?

Reply:
No, this is obsolete and can be deleted